### PR TITLE
Rebustness and :fetch for user repos

### PR DESCRIPTION
**Pull request by gkellogg**
1) Changed definition of config Secrets so that the constant isn't re-defined, or referred to before being defined.
2) Added fetch(:repos) for user, to retrieve users's repositories.
3) Added some robustness of Base#parse_attributes.

Note, you're master branch is still referencing a nonexistent commit, which causes problems when cloning.

### CODE REVIEW

**Comment by farnoy** [comment]
Sorry for such a long time of inactivity! I thought this project was useless to anybody, seeing as it still uses API **v2** and does weird thing so often. I will take a look at this, and will investigate that nonexistent commit, thanks

File: lib/github-api-client/organization.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def fetch(*things)
     private
     def get_members
       members = YAML::load(GitHub::Browser.get "/organizations/#{login}/public_members")['users']
-      puts "Fetching members for #{"organization".color(:magenta).bright} #{self.login.color(:green).bright}"
+      puts "Fetching members for #{"organization".color(:magenta).bright} #{self.login.dup.color(:green).bright}"
```
Comment (farnoy) [inline lib/github-api-client/organization.rb:33]: was .color(:green).bright affecting ActiveRecord attribute `:login`? Is that why #dup was used?

**Comment by farnoy** [comment]
[Merged](https://github.com/farnoy/github-api-client/commit/ea90971f9e4bcb28c16c3438020ae6a328a8540f) into `integration-maint`

Will merge into maintenance branch and be released for version `v0.3.1`, which will be a minor update.

File: lib/github-api-client/organization.rb (hunk 1 above)
Comment (gkellogg) [inline lib/github-api-client/organization.rb:33]: Yes, the attribute value contained the escape characters, which was messing things up. The #dup fixed the problem.

Comment (farnoy) [inline lib/github-api-client/organization.rb:33]: well, then thanks for that one!

**Comment by farnoy** [comment]
Pushed for [v0.3.2](https://rubygems.org/gems/github-api-client/versions/0.3.2)

**Pull request closed without merging**